### PR TITLE
Update improving.html

### PR DESCRIPTION
**Pull request by James-Green**
Minor simplification or clarifications throughout
Typo section 2
Clarified videos are non-technical in section 3
Changed idiom? "from new" to "without this information" in section 4
Removed sentences with series of "examples" in section 4 above list of web a11y eval tools to simplify and shorten
Reworked sentence with "following these explorations" to simplify and clarify in section 4
Replaced one of two "considers" in close proximity in section 5
Replaced 2 of 3 "processes" in close proximity in section 5
Removed "requires less testing with end- users etc. as I suggested in https://github.com/w3c/wai-planning-and-implementation/issues/89
Clarified "target" and multi-tiered approach in section 7 (this removed typo "need to define A")
Changed "aspects" to "tips" since we don't define them as aspects of anything particular
Removed one of two "Other" in close proximity in section 8 and clarified sentence
Simplified last section.
Added "can" to fix typo in last sentence.

### CODE REVIEW

File: improving.html [hunk 1]
```diff
@@ -92,49 +92,49 @@
...
+		<li><a href="https://www.w3.org/WAI/perspectives">Web Accessibility Perspectives</a> &mdash; Short non-technical videos that introduce accessibility features and their benefits for everyone.</li>
 	</ul>
 	<p>If there is a need, more in-depth introductions and background is provided in <a href="https://www.w3.org/WAI/intro/people-use-web/">How People with Disabilities Use the Web</a>. Specifically the <a href="https://www.w3.org/WAI/intro/people-use-web/principles">Accessibility Principles</a> section of the resource introduces the accessibility requirements and guidelines.</p>
 
 	<h2 id="explore">What are the Issues?</h2>
-	<p>If you already know of the specific issues you need to address, you can skip this section. If you are starting to address accessibility from new, the following manual helps you get a rough idea of the current situation on your website:</p>
+	<p>If you already know the specific issues you need to address, you can skip this section. If you are starting to address accessibility without this information, the following manual helps you get a rough idea of the current situation on your website:</p>
```
Comment (nitedog) [inline improving.html]: Maybe "without background" instead? I think "this information" is ambiguous in this sentence.

File: improving.html [hunk 2]
```diff
@@ -92,49 +92,49 @@
...
+	<p>Some web accessibility evaluation tools may help you further explore the accessibility on your website:</p>
 	<ul>
 		<li><a href="https://www.w3.org/WAI/ER/tools/">List of Web Accessibility Evaluation Tools</a> &mdash; Customizable list that allows you to search for different types of tools.</li>
 	</ul>
-	<p>Note that some web accessibility evaluation tools may be complex and confusing to use for people new to accessibility. They typically only check a subset of issues, and may be prone to misleading results.</p>
-	<p>Following these explorations, a more comprehensive evaluation may be needed to get a full picture of the situation. Such thorough evaluations require accessibility expertise. The following resources help you evaluate your website:</p>
+	<p>Note: some web accessibility evaluation tools may be complex and confusing to use for people new to accessibility. They typically only check a subset of issues, and may be prone to misleading results.</p>
+	<p>To get a complete picture of the situation, a comprehensive evaluation will likely be needed. Such thorough evaluations require accessibility expertise. The following resources help you evaluate your website:</p>
```
Comment (nitedog) [inline improving.html]: Change "will likely" to "may"? Some people felt that a full review may be too much in some situations, when there is too little time. Or they may be reacting to a known issue, like a specific complaint - also here they may not need a full review.

**Comment by nitedog** [comment]
Great edits, many thanks! Just two minor responses inline - please check them and let me know what you think.

**Comment by yatil** [comment]
I agree, good changes.

_Side comment: I love the inline comments and the focused discussion this enables :-)_

**Comment by yatil** [comment]
HTML Diff of these changes: http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fw3c.github.io%2Fwai-planning-and-implementation%2Fimproving.html&doc2=https%3A%2F%2Fcdn.rawgit.com%2FJames-Green%2Fwai-planning-and-implementation%2F2239cd8276a0e393ac0e7df607c47819f27e6bb9%2Fimproving.html

File: improving.html (hunk 1 above)
Comment (James-Green) [reply improving.html]: Yeah I went back and forth on that, I originally had "from scratch" but wasn't sure that idiom was used internationally.  I also considered "from the ground up" but felt that sounded like they haven't even built the site, so I think "without background" works. :)

File: improving.html (hunk 2 above)
Comment (James-Green) [reply improving.html]: I am fine with using may if I'm the only one who likes will likely.  My rationale is that having too little time or reacting to a complaint is fine and may be too much for some situations, but if you want a "complete picture" you need a full review and in those situations where it's too much, they will likely not ever have the complete picture.

**Pull request merged**